examples/llava/minicpmv-convert-image-encoder-to-gguf.py

```diff
@@ -158,13 +579,15 @@ def bytes_to_unicode():
 has_text_encoder = True
 has_vision_encoder = True
 has_minicpmv_projector = False
+
 if args.text_only:
     fname_middle = "text-"
     has_vision_encoder = False
 elif args.minicpmv_projector is not None:
     fname_middle = "mmproj-"
     has_text_encoder = False
     has_minicpmv_projector = True
+    minicpmv_version = 3
```

Comment:
Is this line necessary? It overrides `minicpmv_version` value set in the command line when converting MiniCPM-V2.5 which results in a broken mmproj-model-f16.gguf.